src/main/java/com/kalessil/phpStorm/phpInspectionsEA/inspectors/phpDoc/UnknownInspectionInspector.java

```diff
@@ -77,8 +79,7 @@ public void visitPhpDocTag(@NotNull PhpDocTag tag) {
                 if (reported.size() > 0) {
                     final PsiElement target = tag.getFirstChild();
                     if (null != target) {
-                        final String message = "Unknown inspection: %i%".replace("%i%", String.join(", ", reported));
-                        holder.registerProblem(target, message, ProblemHighlightType.WEAK_WARNING);
+                        holder.registerProblem(target, message.replace("%i%", String.join(", ", reported)), ProblemHighlightType.WEAK_WARNING);
```

Comment:
Are you ok with `final String messageFormatted` as message is already part of the class now (`private static final String message = "Unknown inspection: %i%.";`)?